scrapers/BangBros.yml

```
@@ -16,6 +16,7 @@ xPathScrapers:
         Name: //div[@class="vdoCast"]/a[position()>1]/text()
       Image:
         selector: //video/@poster
+        #selector: //img[@id="player-overlay-image"]/@src # Better image but can fail on older scenes
```

Use this:
```suggestion
        selector: //img[@id="player-overlay-image"]/@src|//video/@poster
```

With this, the first selector that is matched will be the image.

```
20-08-10 17:40:58 Debug   [1][Image] = https://x-images5.bangbros.com/bangpov/shoots/bpov17671/members/450x340.jpg
2020-08-10 17:40:58 Debug   [0][Image] = https://x-images5.bangbros.com/bangpov/shoots/bpov17671//big_trailer.jpg
```

and with the selectors swapped:

```
        selector: //video/@poster|//img[@id="player-overlay-image"]/@src
```
```
2020-08-10 17:41:41 Debug   [1][Image] = https://x-images5.bangbros.com/bangpov/shoots/bpov17671//big_trailer.jpg
2020-08-10 17:41:41 Debug   [0][Image] = https://x-images5.bangbros.com/bangpov/shoots/bpov17671/members/450x340.jpg
```

---

Good idea, but i scraped more scenes today and i saw a new problem with this selector.
Sometimes the screen is not correct. 
Exemple: 
`https://bangbros.com/video3405997/elsa-jean-begs-her-stepdad-for-good-sex`

Result:
`img https://x-images4.bangbros.com/bangbrosclips/shoots/bbc15629//big_trailer.jpg`
`@poster https://x-images4.bangbros.com/bangbrosclips/shoots/bbc15629/members/450x340.jpg`

The big trailer is not a correct screenshot for this scene, i don't think it's Elsa Jean and not from this scene.

`//video/@poster` always give the correct screen.